commit 5d69e358de6b0fe5b378327d23ed25dbab53fb63 (refs/changes/50/3950/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-05-14T10:22:27-07:00 (1 year, 5 months ago)
    
    TRITON-382 binder update in MANTA-3676 broken binder usage in Triton: "config.staticLabels.instance (string) is required"

diff --git a/sapi_manifests/binder/template b/sapi_manifests/binder/template
index beb7a06..e3ca8c0 100644
--- a/sapi_manifests/binder/template
+++ b/sapi_manifests/binder/template
@@ -1,7 +1,8 @@
 {
+    {{! "dns_domain" means this is Triton (vs Manta).
+        Only render recursion block in Triton. }}
     {{#dns_domain}}
     "dnsDomain": "{{{dns_domain}}}",
-    "datacenterName": "{{{datacenter_name}}}",
     "recursion": {
         "regionName": "{{{region_name}}}",
         "datacenterName": "{{{datacenter_name}}}",
@@ -22,13 +23,15 @@
             "clientTimeout": 120000,
             "connectTimeout": 3000
         }
-    }
+    },
     {{/dns_domain}}
     {{^dns_domain}}
     "dnsDomain": "{{{DNS_DOMAIN}}}",
-    "datacenterName": "{{{DATACENTER}}}",
+    {{/dns_domain}}
+
+    {{! Metrics labels values. }}
+    "datacenterName": "{{{datacenter_name}}}",
     "instance_uuid": "{{auto.ZONENAME}}",
     "server_uuid": "{{auto.SERVER_UUID}}",
     "service_name": "{{SERVICE_NAME}}"
-    {{/dns_domain}}
-}
+}
\ No newline at end of file
